AlphaWallet/Tokens/ViewControllers/TokensViewController.swift

```diff
@@ -336,11 +338,11 @@ extension TokensViewController: UITableViewDataSource {
             return cell
         case .erc721:
             let cell = tableView.dequeueReusableCell(withIdentifier: NonFungibleTokenViewCell.identifier, for: indexPath) as! NonFungibleTokenViewCell
-            cell.configure(viewModel: .init(token: token))
+            cell.configure(viewModel: .init(token: token, chainId: session.config.chainID))
```

Comment:
Can we pass in session.config.server instead of session.config.chainID. Stronger type check, etc.